playground/deepspeed_config_s2.json

```diff
@@ -1,12 +1,34 @@
 {
-  "zero_optimization": {
-    "stage": 2,
-    "offload_optimizer": {
-      "device": "cpu"
+  "bf16": {
+    "enabled": "auto"
+  },
```

Comment:
I'm a little worried about including this into the playground file because not all GPUs support bf16; I think it would be best to remove this for generalizability.

Reply:
I have reset the config to what is on your repo.